pkg/services/licensing/oss.go

```diff
@@ -7,6 +7,10 @@ import (
 	"github.com/grafana/grafana/pkg/setting"
 )
 
+const (
+	OpenSource = "Open Source"
```

Comment:
Please un-export this one, since it isn't used outside of the package.
```suggestion
	openSource = "Open Source"
```